lib/pages/themes-page.js

```diff
@@ -52,28 +46,12 @@ export default class ThemesPage extends BaseContainer {
 	}
 
 	async waitUntilThemesLoaded() {
-		const selector = by.css( '.themes-list .is-placeholder' );
-		const self = this;
-		return await self.driver.wait(
-			async function() {
-				return await driverHelper
-					.isElementPresent( self.driver, selector )
-					.then( function( present ) {
-						return ! present;
-					} );
-			},
-			this.explicitWaitMS,
-			'The themes are still loading after waiting for them'
-		);
+		return driverHelper.waitTillNotPresent( this.driver, by.css( '.themes-list .is-placeholder' ) );
```

Comment:
We should use `await` for driverHelper methods i.e. for every method which returns promise. This is not mandatory for now since Promise manager is enabled and once we disable it we need to check all our methods once again.

Reply:
Thanks, missed that one